### PR TITLE
Validate application.properties: unknown properties

### PR DESCRIPTION
**Pull request by angelozerr**
Validate application.properties: unknown properties

See https://github.com/redhat-developer/quarkus-ls/issues/26

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -129,13 +129,39 @@
...
+				},
+		        "quarkus.tools.validation.unknown.severity": {
+						  "type": "string",
+		          "enum": [
+		            "none",
+		            "warning",
+		            "error"
+		          ],
+		          "default": "warning",
+		          "description": "Unknown Quarkus properties severity validation.",
```
Comment (xorye) [inline package.json]: I think "Validation severity for unknown Quarkus properties" sounds a bit better.

File: package.json [hunk 2]
```diff
@@ -129,13 +129,39 @@
...
+		          "scope": "window"
+		        },
+		        "quarkus.tools.validation.unknown.excluded": {
+		          "type": "array",
+		          "default": [],
+		          "items": {
+		            "type": "string"
+		          },
+		          "description": "Array of properties to ignore by Unknown Quarkus properties validation.",
```
Comment (xorye) [inline package.json]: "Array of properties to ignore for unknown Quarkus properties validation."

Is there a reason why "Unknown" was capitalized?

File: package.json (hunk 1 above)
Comment (angelozerr) [reply package.json]: @xorye updated

File: package.json (hunk 2 above)
Comment (angelozerr) [reply package.json]: @xorye updated

**Comment by angelozerr** [comment]
Thank's @xorye for your review.

**Pull request merged**